finder-app/finder.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+if [ $# -lt 2 ]
+then
+	echo "Insufficient number of arguments provided"
```

Comment:
@ECleverito think about how you could make the error message more descriptive to help the user do the right thing on the next call...